experiments/LES/surfacebubble.jl

```diff
@@ -0,0 +1,238 @@
+using Distributions
+using Random
+using StaticArrays
+using Test
+using DocStringExtensions
+using LinearAlgebra
+
+using CLIMA
+using CLIMA.Atmos
+using CLIMA.GenericCallbacks
+using CLIMA.DGmethods.NumericalFluxes
+using CLIMA.LowStorageRungeKuttaMethod
+using CLIMA.Mesh.Filters
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+using CLIMA.VariableTemplates
+
+import CLIMA.DGmethods: boundary_state!
+import CLIMA.Atmos: atmos_boundary_state!, atmos_boundary_flux_diffusive!, flux_diffusive! 
+import CLIMA.DGmethods.NumericalFluxes: boundary_flux_diffusive!
+
+
+# -------------------- Surface Driven Bubble ----------------- # 
+# 1) Boundary Conditions: (Closed box system)
+#      Momentum: No flow penetration [NoFluxBC()]
+#      Energy:   Spatially varying non-zero heat flux up to time t₁
+# 2) Domain: 2500[m] × 2500[m] × 5000[m]
+# ------------------------------------------------------------ # 
+
+"""
+  SurfaceDrivenBubbleBC <: BoundaryCondition
+Y ≡ state vars
+Σ ≡ diffusive vars
+A ≡ auxiliary vars
+⁺ and ⁻ refer to exterior, interior faces
+
+# Fields
+$(DocStringExtensions.FIELDS)
+"""
+struct SurfaceDrivenBubbleBC{FT} <: BoundaryCondition
+  "Prescribed MSEF Magnitude [W/m^2]"
+  F₀::FT
+  "Spatial Parameter [m]"
+  σ::FT
+  "Surface Heater Radius [m]"
+  a::FT
+  "Surface Heater Center [m]"
+  x₀::FT
+  "Time Cutoff [s]"
+  t₁::FT
+end
+function atmos_boundary_state!(nf::Union{NumericalFluxNonDiffusive,NumericalFluxGradient}, 
+                               bc::SurfaceDrivenBubbleBC, 
+                               m::AtmosModel,
+                               Y⁺::Vars, A⁺::Vars, 
+                               n⁻, 
+                               Y⁻::Vars, A⁻::Vars, 
+                               bctype, t,_...)
+  FT = eltype(Y⁺)
+  @inbounds begin
+    # Momentum b.c. prescribed (no flow across wall boundary)
+    if typeof(nf) <: NumericalFluxNonDiffusive
+      Y⁺.ρu -= 2 * dot(Y⁻.ρu, n⁻) * SVector(n⁻)
+    else
+      Y⁺.ρu -=  dot(Y⁻.ρu, n⁻) * SVector(n⁻)
+    end
+  end
+  nothing
+end
+function atmos_boundary_flux_diffusive!(nf::CentralNumericalFluxDiffusive, 
+                                        bc::SurfaceDrivenBubbleBC,
+                                        m::AtmosModel, 
+                                        F,
+                                        Y⁺::Vars, Σ⁺::Vars, A⁺::Vars, 
+                                        n⁻, 
+                                        Y⁻::Vars, Σ⁻::Vars, A⁻::Vars, 
+                                        bctype, t, _...)
+  FT = eltype(Y⁻)
+  @inbounds begin
+    # Momentum b.c. prescribed (no flow across wall boundary)
+    if typeof(nf) <: NumericalFluxNonDiffusive
+      Y⁺.ρu -= 2 * dot(Y⁻.ρu, n⁻) * SVector(n⁻)
+    else
+      Y⁺.ρu -=  dot(Y⁻.ρu, n⁻) * SVector(n⁻)
+    end
+  end
```

Comment:
```julia
    atmos_boundary_state!(nf, NoFluxBC(), m,
                          Y⁺, Σ⁺, A⁺, n⁻,
                          Y⁻, Σ⁻, A⁻,
                          bctype, t)
```
similar to [dycoms](https://github.com/climate-machine/CLIMA/blob/c688b17762e3ef60233a46f6ba3fe094aa3bc2f6/experiments/LES/dycoms.jl#L89-L92)

Reply:
Noted - will use the `NoFluxBC` default